kubectl-fdb/main.go

```diff
@@ -31,7 +31,7 @@ import (
 func main() {
 	flags := pflag.NewFlagSet("kubectl-fdb", pflag.ExitOnError)
 	pflag.CommandLine = flags
-
+	cmd.PluginVersionChecker = &cmd.RealVersionChecker{}
```

Comment:
See my comment above, we should pass this down to the `NewRootCmd` and not define a global variable.